### PR TITLE
feat(rxPaginate): Add ability to disable itemsPerPage persistence

### LINKED ISSUE #1074: Add ability to disable rxPaginate persistence
For an upcoming change to Encore Account Summary, we will have paginated data with page sizes 5, 10, and 20.  However, when the user selects "20" this page size should NOT persist to paginated data in other locations within Encore.

Please add the ability to disable persistence for a single paginated data set.

I'd suggest something like a `persist` attribute that defaults to `true`. When `persist="false"`, changing the page size should NOT persist across URL locations.

### PR DESCRIPTION
**Pull request by parlarjb**
FRMW-55
Closes #1074

### CODE REVIEW

File: src/rxPaginate/templates/rxPaginate.html [hunk 1]
```diff
@@ -32,7 +32,7 @@
             <div>
                 Show
                 <ul>
-                    <li ng-repeat="i in pageTracking.itemSizeList"><button ng-disabled="pageTracking.isItemsPerPage(i)" class="pagination-per-page-button" ng-disabled="i == pageTracking.itemsPerPage" ng-click="updateItemsPerPage(i)">{{ i }}</button> </li>
+                    <li ng-repeat="i in pageTracking.itemSizeList"><button ng-disabled="pageTracking.isItemsPerPage(i)" class="pagination-per-page-button" ng-disabled="i == pageTracking.itemsPerPage" ng-click="pageTracking.setItemsPerPage(i)">{{ i }}</button> </li>
```
Comment (parlarjb) [inline src/rxPaginate/templates/rxPaginate.html]: This is the key change. Previously we were using a "global" method to set the new items per page. Now it's been moved to a method on the individual pager instance.

**Comment by megalord** [comment]
LGTM

File: src/rxPaginate/rxPaginate.js [hunk 2]
```diff
@@ -452,17 +427,44 @@ angular.module('encore.ui.rxPaginate', ['encore.ui.rxLocalStorage', 'debounce'])
 
     }
 
-    return {
+    var PageTracking = {
+        /**
+        * @property {number} itemsPerPage This is the current setting for the number
+        * of items to display per page.
+        * @property {boolean} [persistItemsPerPage=false] Whether or not a change to this pager's itemsPerPage
```
Comment (CITguy) [inline src/rxPaginate/rxPaginate.js]: This should be `[persistItemsPerPage=true]` to match the default value.

Comment (parlarjb) [inline src/rxPaginate/rxPaginate.js]: :facepalm:

**Comment by CITguy** [comment]
LGTM

**Pull request merged**